examples/basics/quick_start.ipynb

```diff
@@ -0,0 +1,195 @@
+{
```

Comment:
<p>"install the labelbox library" -&gt; "install the <code>labelbox</code> library"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>